cmd/experimental/kueue-viz/frontend/Dockerfile

```diff
@@ -1,5 +1,5 @@
 #Build the React app
-FROM node:23 AS build
+FROM registry.redhat.io/rhel9/nodejs-22 AS build
```

Comment:
Can we create a separate Dockerfile.rhel instead of touching the upstream one?

Reply:
Fixed